README.md

```diff
@@ -25,10 +26,53 @@ to nextstrain.org by running:
 nextstrain build . deploy_all
 ```
 
+## H5N1 Cattle Outbreak (2024)
+
+We produce per-segment and whole-genome (concatenated segments) builds for the ongoing H5N1 cattle-flu outbreak.
+These use NCBI data including consensus genomes and SRA data assembled via the Andersen lab's [avian-influenza repo](https://github.com/andersen-lab/avian-influenza).
+
+> It's important to remove any files in `./data` before running these builds otherwise Snakemake may use existing GENBANK data from a previous analysis.
```

Comment:
any time i see a warning like this, i wonder if it would be better handled in the code layer instead. (i.e., have a `rm -rf data` step in the Snakemake.)

Reply:
Reworded this message. Snakemake actually does the right thing, so this isn't actually needed.